template/default.tmpl

```diff
@@ -1,5 +1,5 @@
 {{ define "__alertmanager" }}AlertManager{{ end }}
-{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts{{ end }}
+{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}{{ end }}
```

Comment:
The template sends out receiver strings for a firing/resolved alert as configured in the alertmanager.yml config file. These are not influenced by using regexp's on the front end for filtering, so I don't think this is necessary.